test/bacnet/include/bacport.h

```diff
@@ -0,0 +1,38 @@
+/**************************************************************************
```

Comment:
@Michail-Antropov When is this file version used instead of the one under `ports/`?

Reply:
Unfortunately, ztest does not have access to the ports/